h2/stream.py

```diff
@@ -723,6 +724,7 @@ def send_headers(self, headers, encoder, end_stream=False):
         Returns a list of HEADERS/CONTINUATION frames to emit as either headers
         or trailers.
         """
+
```

Comment:
The general style in this codebase is to not have a blank line following a docstring. =)